.travis.yml

```diff
@@ -48,7 +49,7 @@ jobs:
         - aws s3 sync $TRAVIS_BUILD_SHARED_STORAGE $AWS_S3_BUILD_SHARED_STORAGE
     - stage: e2e_tests
       name: Execute e2e tests
-      if: branch = master OR type = pull_request OR tag =~ /^release.*$/
+      if: branch = master OR tag =~ /^release.*$/

```

Comment:
it should run on all branches not only on master if you remove PR

Reply:
Good point.

Reply:
Fixed.